util/network-devp2p/src/discovery.rs

```diff
@@ -42,9 +42,9 @@ const PACKET_PONG: u8 = 2;
 const PACKET_FIND_NODE: u8 = 3;
 const PACKET_NEIGHBOURS: u8 = 4;
 
-const PING_TIMEOUT: Duration = Duration::from_millis(300);
+const PING_TIMEOUT: Duration = Duration::from_millis(500);
```

Comment:
From my point of view half a second does not seems a lot, and it is the same as geth, probably fine.

Reply:
From the specs (https://github.com/ethereum/devp2p/blob/master/rlpx.md#node-discovery): _request timeouts are 300ms_ ; but I agree it' pretty low, that's why the 500ms